chatkit-core/src/test/kotlin/com/pusher/chatkit/UserTestUtil.kt

```diff
@@ -1,10 +1,7 @@
 package com.pusher.chatkit
 
-import com.pusher.chatkit.users.User
-import com.pusher.chatkit.util.dateFormat
-import java.util.Date
-
-fun simpleUser(id: String): User {
-    val createdUpdatedAt = dateFormat.format(Date())
-    return User(id, createdUpdatedAt, createdUpdatedAt, null, null, null)
-}
+fun simpleUser(id: String) {}
+//        : User {
```

Comment:
here I think we should use UserApiType, the `fun` can stay as it will be useful for functional tests in the future

Reply:
👍 done!